trino/client.py

```diff
@@ -244,6 +247,12 @@ def __init__(
             self._auth.set_http_session(self._http_session)
             self._exceptions += self._auth.get_exceptions()
 
+        if extra_credential is not None and len(extra_credential) > 0:
```

Comment:
Add `extra_credential` to ClientSession and fix `http_headers()` method for consistency.